.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name != 'pull_request' }}
+    steps:
+      - uses: actions/checkout@v2
+        name: Checkout [main]
+        with:
+          fetch-depth: 0
+      - run: npm install
+      - run: npm run build:elements
```

Comment:
I don't see a logic to create a Release. In general, it is ok with me to create a Release manually, but I would like an artefact (zip archive with JS and CSS) to be published by Job at least (https://docs.github.com/en/actions/advanced-guides/storing-workflow-data-as-artifacts).

Reply:
Yep that's true, I'll work on that but I had to create a PR first to check if the git actions are triggered and running successfully :-)

Reply:
@01sas fixed. The main build will now create the artifact, which will be zipped when downloaded from git by default see https://github.com/actions/upload-artifact#zipped-artifact-downloads